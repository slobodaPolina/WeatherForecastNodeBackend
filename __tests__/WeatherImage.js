import React from 'react';
import renderer from 'react-test-renderer';
import WeatherImage from '../src/components/WeatherImage';

it('renders WeatherImage correctly', () => {
    const tree = renderer.create(
        <WeatherImage name="Clear"/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
