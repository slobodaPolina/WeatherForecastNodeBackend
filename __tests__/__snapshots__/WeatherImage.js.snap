// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WeatherImage correctly 1`] = `
<img
  alt="Weather"
  className="weatherImage"
  src="/images/Clear.jpg"
/>
`;
